### ISSUE TITLE
Fix erroneous Maven build with "geoserver" profile

### ISSUE THREAD

**Issue by seancrow** [geosolutions-it/csi-sira#287]
- [ ] fix erroneous `Maven` build with "`geoserver`" profile, which fails with the following error:

> $ mvn clean install -Pgeoserver -DskipTests
> [INFO] Scanning for projects...
> [ERROR] [ERROR] Some problems were encountered while processing the POMs:
> [FATAL] Non-resolvable parent POM for org.geoserver.web:gs-web-app:2.8.x-SIRA-SNAPSHOT: Could not find artifact org.geoserver:gs-web:pom:2.8-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 10, column 11
> [ERROR] The build could not read 1 project -> [Help 1]
> [ERROR]
> [ERROR]   The project org.geoserver.web:gs-web-app:2.8.x-SIRA-SNAPSHOT (C:\work\csi\projects\SIPRA\eclipse\sipradec\workspaces\sira-root\geoserver\src\web\app\pom.xml) has 1 error
> [ERROR]     Non-resolvable parent POM for org.geoserver.web:gs-web-app:2.8.x-SIRA-SNAPSHOT: Could not find artifact org.geoserver:gs-web:pom:2.8-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 10, column 11 -> [Help 2]

**Comment by seancrow** [comment]
@diegosanmartino and @fra1312, the error occurs only when building against "`geoserver`" profile, while building succeds when using the other profile, "`web`"

**Comment by seancrow** [comment]
Hi @mbarto: would you please take a look at this error during `Maven` building?

**Comment by seancrow** [comment]
Hi @mbarto: it seems this is the offending instruction in module `geoserver/src/web/app` `Maven` `POM`:

```
>   <parent>
>     <groupId>org.geoserver</groupId>
>     <artifactId>gs-web</artifactId>
>     <version>2.8-SNAPSHOT</version>
>   </parent>
```

**Comment by seancrow** [comment]
Closing issue, solved with #317 by @mbarto
